### PR TITLE
OLS-1167: Document that OLS does not have update path for existing installa…

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Affects:
[lightspeed-main](https://github.com/openshift/openshift-docs/tree/lightspeed-docs-main)
[lightspeed-docs-1.0tp1](https://github.com/openshift/openshift-docs/tree/lightspeed-docs-1.0tp1)

This PR is part of the standalone doc set for the Lightspeed project. Kathryn is aware that this content applies for a product that is part of a Technology Preview release. The project is seeking feedback from early adopters.

PR must be CP'd back to the tp1 branch.

Version(s): Tech Preview

Issue: https://issues.redhat.com/browse/OLS-1167

Link to docs preview: https://83721--ocpdocs-pr.netlify.app/openshift-lightspeed/latest/release_notes/ols-release-notes.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by JoaoFula** [comment]
LGTM

File: modules/ols-0-2-0-release-notes.adoc [hunk 1]
```diff
@@ -14,6 +14,8 @@ The following enhancements have been made with {ols-offical} 0.2.0:
 
 * This release introduces the {ols-offical} Operator as a Technology Preview release. Previously, the {ols-short} Operator was a Developer Preview release. 
 
+* Existing Developer Preview installations of {ols-offical} version 0.1.6 and earlier that use the preview {olm} (OLM) channel cannot update to the 0.2.0 Technology Preview version. You must switch the update channel to *alpha* and then install the 0.2.0 Technology Preview version.
```
Comment (xrajesh) [inline modules/ols-0-2-0-release-notes.adoc:17]: Existing Developer Preview installations of {ols-offical} version 0.1.6 and earlier that use the preview {olm} (OLM) channel must switch the update channel to *alpha* and then install the 0.2.0 Technology Preview version. 
@rh-tokeefe  I would move to one single line like above, look better

Comment (rh-tokeefe) [reply modules/ols-0-2-0-release-notes.adoc:17]: Thanks for the suggestion. Previously I thought the main point we are trying to convey is that there is no direct upgrade path. I like this change. Done.

**Comment by stevsmit** [comment]
LGTM

**Pull request merged**